apps/cli/src/commands/newGetDeviceInfo.ts

```diff
@@ -0,0 +1,32 @@
+import { getDeviceInfoAction } from "@ledgerhq/live-common/deviceSDK/actions/getDeviceInfo"; 
+import { Observable } from "rxjs";
+import { deviceOpt } from "../scan";
+
+export default {
```

Comment:
this CLI should be deleted ;) (i renamed it `deviceSDKGetDeviceInfo` already)

Reply:
Done